google-cloud-storage/src/test/java/com/google/cloud/storage/ParallelCompositeUploadBlobWriteSessionConfigTest.java

```diff
@@ -84,7 +84,7 @@ public void partNameStrategy_objectNamePrefix() throws Exception {
         // name digest
         () -> assertField(fmt, 1).hasLength(22),
         () -> assertField(fmt, 2).isEqualTo("0001-0096.part"),
-        () -> assertThat(fmt).startsWith("a/b/obj/"));
+        () -> assertThat(fmt).startsWith("a/b/obj"));
```

Comment:
Why did this need to change?

Reply:
this genuinely needed to change as the prefix doesn't create a "directory". the name is just slapped on the front of the part. both of these failing tests somehow got missed.